.rubocop-http---shopify-github-io-ruby-style-guide-rubocop-yml

```diff
@@ -1080,9 +1080,6 @@ Performance/FixedSize:
 Performance/FlatMap:
   EnabledForFlattenWithoutParams: false
 
-Performance/HashEachMethods:
```

Comment:
Hmm, what does this removal do?

Reply:
Actually, I don't know Jeff, every time I change code this file changes automatically. Can you specify the purpose of this file? Thanks.

Reply:
Ah -- @ryzokuken @siaw23 do you know why?